controllers/operands/kubevirt.go

```diff
@@ -880,8 +878,19 @@ func getMandatoryKvFeatureGates(isKVMEmulation bool) []string {
 // get list of feature gates or KV FG list
 func getKvFeatureGateList(fgs *hcov1beta1.HyperConvergedFeatureGates) []string {
 	checks := getFeatureGateChecks(fgs)
+
 	res := make([]string, 0, len(checks)+len(mandatoryKvFeatureGates)+1)
-	res = append(res, mandatoryKvFeatureGates...)
+
+	if fgs.DownwardMetrics != nil && !*fgs.DownwardMetrics {
```

Comment:
Please revert. This function sould not be changed. Please follow one of the other optional featur gates. It should be in getFeatureGateChecks function.

Reply:
Done.